dist/Leaflet.fullscreen.js

```diff
@@ -1,152 +1,170 @@
-L.Control.Fullscreen = L.Control.extend({
-    options: {
-        position: 'topleft',
-        title: {
-            'false': 'View Fullscreen',
-            'true': 'Exit Fullscreen'
+(function (factory) {
+    var L;
```

Comment:
Do we need this variable or can the Node/CommonJS case just do `module.exports = factory(require('leaflet'))`?